app/src/main/java/service/RetrofitClientInstance.kt

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
I commented out this class, you have this retrofitClientInstance in a different place as well.

Reply:
Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.